ml-testing-toolkit/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 description: ml-testing-toolkit Helm chart for Kubernetes
 name: ml-testing-toolkit
 version: 15.4.0
```

Comment:
Why are we not bumping?

Reply:
This was pre-emptively bumped in the branch I'm merging to. So I'd be double bumping here.